.github/workflows/tag-version.yml

```diff
@@ -18,15 +18,15 @@ jobs:
           fetch-depth: 0 # fetch all tags
 
       - name: Tag and push if new version
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Q: Do we set a service token from openproject-ci or a personal token? I've found the former to be more future proof

Reply:
It's a github thing that's available by default for actions (if I understood correctly) https://docs.github.com/en/actions/tutorials/authenticate-with-github_token

So it's not really a provided token, more like internals of github actions.